.travis.yml

```diff
@@ -11,7 +11,6 @@ cache: bundler
 
 before_install:
   - docker-compose -f docker-compose-travis.yml up -d
-  - docker exec forum_testing chmod +x /edx/app/forum/cs_comments_service/.travis/run_tests.sh
```

Comment:
I removed this because I just changed the default permission of the file so it is executable.  Then you don't have to do this.